.github/workflows/pr_checklist.yml

```diff
@@ -0,0 +1,33 @@
+name: PR Checklist
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  add-checklist:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Comment PR with checklist
+        uses: peter-evans/create-or-update-comment@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            Thank you for submitting your PR. The PR states are In progress (or Draft) -> Tech review -> Doc review -> Editorial review -> Merged.
```

Comment:
Should these match the GitHub labels?

Reply:
> Some of them do (like "In progress", "Doc review", and "Tech review"). "Editorial review" label is "Final editorial review" but I'd rather change the label if needed because the goal here is to provide some hints with minimum number of words.

Agreed - I changed the label.